bin/ch/ch.cpp

```diff
@@ -671,6 +671,10 @@ HRESULT ExecuteTest(const char* fileName)
         }
     }
 Error:
+    if( fileContents != nullptr)
+    {
+        free((char*)fileContents);
```

Comment:
The bytecode buffers are being passed in as just byte*s, not as ExternalArrayBuffers, so we can't register the callback that way, as far as I can tell.